driver-portal/src/ClientApp/src/app/letters-to-driver/letters-to-driver.component.ts

```diff
@@ -59,9 +61,21 @@ export class LettersToDriverComponent implements OnInit {
     this.filteredDocuments = this._letterDocuments?.slice(0, this.pageSize);
   }
 
+  letterOutDocuments: Document[] = [];
+
   ngOnInit(): void {
     window.scrollTo(0, 0);
+    if(this.loginService.userProfile?.id){
```

Comment:
and then with userService this code will be the same and behind the scenes userService will use this code

Reply:
I have adjusted the code and pushed the changes.